docs/content/specs/shared/_index.md

```diff
@@ -494,10 +494,10 @@ Only the AVM core team can grant permissions to the [BRM](https://aka.ms/BRM) re
 
 Module owners **MUST** notify the AVM core team, when the `-module-owners-` and `-module-owners-` teams are created, so the AVM core team can grant permissions to the BRM repo and make the necessary changes to the `CODEOWNERS` file.
 
-| GitHub Team Name                                        | Description                                                                   | Permissions | Where to work?          |
-|---------------------------------------------------------|-------------------------------------------------------------------------------|-------------|-------------------------|
-| `@azure/<dashed module name>-module-owners-bicep`       | Modules Owners of the <module name> AVM Bicep <resource/pattern> module       | **Write**   | Need to work in a fork. |
-| `@azure/<dashed module name>-module-contributors-bicep` | Modules Contributors of the <module name> AVM Bicep <resource/pattern> module | **Triage**  | Need to work in a fork. |
+| GitHub Team Name                                            | Description                                                                  | Permissions | Permissions granted through                                           | Where to work?          |
+|-------------------------------------------------------------|------------------------------------------------------------------------------|-------------|-----------------------------------------------------------------------|-------------------------|
+| `@Azure/<hyphenated module name>-module-owners-bicep`       | Module Owners of the <module name> AVM Bicep <resource/pattern> module       | **Write**   | Assignment to the `@Azure/avm-technical-reviewers-bicep` parent team. | Need to work in a fork. |
+| `@Azure/<hyphenated module name>-module-contributors-bicep` | Module Contributors of the <module name> AVM Bicep <resource/pattern> module | **Triage**  | `@Azure/avm-module-contributors-bicep` parent team.                   | Need to work in a fork. |
```

Comment:
Before approving, the internal discussion on this topic (permissions required by -contributors- teams) has to be concluded first.